flink-rpc/flink-rpc-akka/src/main/resources/META-INF/NOTICE

```diff
@@ -8,25 +8,24 @@ This project bundles the following dependencies under the Apache Software Licens
 
 - com.hierynomus:asn-one:0.5.0
 - com.typesafe:config:1.4.2
-- com.typesafe:ssl-config-core_2.12:0.4.3
-- com.typesafe.akka:akka-actor_2.12:2.6.20
-- com.typesafe.akka:akka-remote_2.12:2.6.20
-- com.typesafe.akka:akka-pki_2.12:2.6.20
-- com.typesafe.akka:akka-protobuf-v3_2.12:2.6.20
-- com.typesafe.akka:akka-slf4j_2.12:2.6.20
-- com.typesafe.akka:akka-stream_2.12:2.6.20
+- com.typesafe:ssl-config-core_2.12:0.6.1
```

Comment:
Were these other dependencies updated as transitive dependencies of Pekko? :thinking:

Reply:
yes. Pekko 1.0.0 is not a pure fork of Akka 2.6.20 unfortunately.